app/hooks/useLazyMint.tsx

```
@@ -0,0 +1,69 @@
+      });
+
+      try {
+        setIsProcessing(true);
+        const { transactionHash } = await sendTransaction({
+          transaction,
+          account: activeAccount!,
+        });
+
+        setTxnHash(transactionHash);
+        setIsProcessing(false);
+      } catch (err: any) {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)